Sources/DeckUI/Views/Presenter.swift

```diff
@@ -115,88 +116,18 @@ public struct Presenter: View {
             .onEnded { value in
                 let tolerance: ClosedRange<CGFloat> = -100...100
                 switch(value.translation.width, value.translation.height) {
-                case (tolerance, ...0):  lineUp()
-                case (tolerance, 0...):  lineDown()
-                case (...0, tolerance):  nextSlide()
-                case (0..., tolerance):  previousSlide()
+                case (tolerance, ...0):  NotificationCenter.default.post(name: .keyUp, object: nil)
+                case (tolerance, 0...):  NotificationCenter.default.post(name: .keyDown, object: nil)
```

Comment:
Would it make sense to also move these `NotificationCenter.default.post` into a `lineUp()` and `lineDown()` in `PresentationState`? 🤔

Reply:
I'd say maybe make it a TODO since each individual slide has its own highlighted line number and that scope creep might get dicey